gsad/src/gsad_http.c

```diff
@@ -1035,6 +1039,25 @@ add_cors_headers (http_response_t *response)
   }
 }
 
+/**
+ * @brief Add header to manage caching to a HTTP response.
+ *
+ * @param[in]  response       The HTTP response to add the headers to.
+ * @param[in]  allow_caching  1 to allow caching, 0 to forbid.
+ */
+void
+add_caching_headers (http_response_t *response, int allow_caching)
+{
+  if (allow_caching == FALSE)
```

Comment:
Would be better to avoid this check here. Either call this function and add the header or don't call the function at all.